scripts/pi-hole/js/db_graph.js

```diff
@@ -215,24 +219,38 @@ $(document).ready(function() {
                                          "-" +
                                          padNumber(time.getMonth()+1) +
                                          "-" +
-                                         padNumber(time.getDate()) +
-                                         " " +
-                                         padNumber(time.getHours()) +
+                                         padNumber(time.getDate());
+                        var until_time = padNumber(time.getHours()) +
                                          ":" +
                                          padNumber(time.getMinutes()) +
                                          ":" +
                                          padNumber(time.getSeconds());
-                        return "Queries from " + from_date + " to " + until_date;
+
+                        if (from_date === until_date) {
+                            // Abbreviated form for intervals on the same day
+                            // We split title in two lines on small screens
+                            if ($(window).width() < 992) {
+                                until_time += "\n";
+                            }
+                            return ("Queries from " + from_time + " to " + until_time + " on " + from_date).split("\n ");
+                        }
+                        // Full tooltip for intervals spanning more than one day
+                        // We split title in two lines on small screens
+                        if ($(window).width() < 992) {
```

Comment:
Instead of checking ``$(window).width() < 992`` twice, you could use a variable for an optional line break, like this:
```js
// We split title in two lines on small screens
var opt_break = "";
if ($(window).width() < 992) {
    opt_break = "\n";
}

// Abbreviated form for intervals on the same day
if (from_date === until_date) {
    return ("Queries from " + from_time + opt_break +" to " + until_time + " on " + from_date).split("\n ");
}
// Full tooltip for intervals spanning more than one day
return ("Queries from " + from_date + opt_break + " " + from_time +
        " to " + until_date + " " + until_time).split("\n ");
```